tests/test-distribution/src/test/java/org/eclipse/jetty/tests/distribution/DistributionTests.java

```diff
@@ -304,4 +307,104 @@ public void testLog4j2ModuleWithSimpleWebAppWithJSP() throws Exception
             IO.delete(jettyBase.toFile());
         }
     }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"http", "https"})
+    public void testWebsocketClientInWebappProvidedByServer(String scheme) throws Exception
```

Comment:
I cannot comment on the `keystore` file, but I don't think it's necessary.
The keystore is copied from module `ssl`, no? The one you added here is not used at all. If that's the case, remove it.